File: Domain/ProjectTask.cs
using System;

namespace Domain
{
    public class ProjectTask
    {
        public int Id { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
        public int ProjectId { get; set; }
        public string ItemName { get; set; }
        public string ItemDescription { get; set; }
        public string ItemCategory { get; set; }
        public decimal UnitRate { get; set; }
        public decimal OrderQty { get; set; }
        public decimal ClaimedQty { get; set; }
        public decimal CurrentValue { get; set; }
        public string Remark { get; set; }
    }
}

File: client-app/src/app/models/projecttask.ts
export interface IProjectTask {
    id: string;
    createdAt: string;
    updatedAt: string;
    projectId: string;
    itemName: string; 
    itemDescription: string;
    itemCategory: string;
    unitRate: number;
    orderQty: number;
    claimedQty: number;
    currentValue: number;
    remark: string;
}